src/python/test/test_nextflow.py

```diff
@@ -103,11 +115,27 @@ def test_src_profile(self):
     @unittest.skipIf(USING_PYTHON2,
         'Skipping as the Nextflow template from which applets are built is for Py3 interpreter only')
     def test_src_inputs(self):
-        src = get_nextflow_src(custom_inputs=[input1, input2])
-        self.assertTrue("if [ -n \"${}\" ];".format(input2.get("name")) in src)
-        self.assertTrue("--{}=${}".format(input2.get("name"), input2.get("name")))
```

Comment:
Seems like we are not asserting anything here

Reply:
@mhrvol I saw we have some tests https://github.com/dnanexus/nextaur-app/pull/11. Is any of them you think shall be included?

Reply:
I've used multiple of these tests when debugging some changes in nextaur, I dont we need we need most of them. There are some for errorStrategy testing, but Jana from QE is creating test suite with these tests (or similar ones)